### ISSUE TITLE
неспроста

### ISSUE THREAD

**Issue by arysin** [brown-uk/dict_uk#252]
спроста і неспроста подати як :coll. На неспроста є купа прикладів у ГРАКу, зокрема з діаспорних видань

**Comment by stjoanis** [comment]
First of all, I must thank you for your project, my wife is working on Ukrainian verbs and I'm trying to make things for her.
So then I have a few questions :
What's the Graddle environment ? I suppose the code is in Java.
I tried to make this project work on CentOS-7 but got a Java UnsupportedClassVersionError
`# gradlew expand
`It's a JNI error during the task Autogen

As I'm not ease with Java I switched to Windows.
But I did not find the bin/expand_win.sh
Anyway I launched gradlew.bat that brought in some Java env, it seems.
But I feel totally lost with no idea what to do and what to expect as a result.

Finally, my wife told me that she's trying to find lemmas like this one ->
близнюк /n20.a.p.ke.<

Does the file out/lemmas.txt contains such information ?
If yes, then I'd like to generate it, or find a common version of it.
I found the dict_corp_vis.txt.bz2 which seems a comprehensive data and my guess is that this project aims at generating lang data such as this one but maybe in a custom format to easily use for particular cases.
Many thanks if someone can guide me to get a way around.

Emmanuel
